GeorgiyYegorashchenko/HW5/Task1.py

```diff
@@ -0,0 +1,5 @@
+# Task 1
+
```

Comment:
good job
look at this version of code
list_of_number = [2, 1, 4, 6, 1, 4, 8, 6]

for i in range(len(list_of_number)):
    list_of_number[i] = float(list_of_number[i])
print(list_of_number)